.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,30 @@
+name: EuroPython website
+
+on:
+  push:
+    branches:
+    - master
+    - dev/ep*
```

Comment:
Could you add `ep20*` to the branches and pull request triggers? We're going to move to `ep2020` branch soon - and probably just keep the `ep20XX` branches in the future, I don't think we should keep a separate `dev/20xx` branch after stopping using the current one.